qt/scientific_interfaces/Indirect/JumpFitModel.cpp

```diff
@@ -243,8 +243,7 @@ void JumpFitModel::addWorkspace(const std::string &workspaceName) {
 
   const auto hwhmWorkspace =
       createHWHMWorkspace(workspace, name, parameters.widthSpectra);
-  IndirectFittingModel::addWorkspace(hwhmWorkspace->getName(),
-                                     Spectra(createSpectra(spectrum.get())));
+  IndirectFittingModel::addWorkspace(hwhmWorkspace->getName(), Spectra(""));
```

Comment:
What is the reasoning behind this change? I think adding a worspace to the fitting data within any Spectra specified may be at least partially responsible for one of the issues below.

Reply:
When the workspace is added initially it adds all the spectra to the spectra list in the model. Previously this was replaced when a specra was added, but with the method now appending spectra to the list without this change it loads all the sectra as well as a double of the one you wanted.